azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -507,7 +506,10 @@ def on_method_request_received(self):
         """The handler function or coroutine that will be called when a method request is received.
 
         The function or coroutine definition should take one positional argument (the
-        :class:`azure.iot.device.MethodRequest` object)"""
+        :class:`azure.iot.device.MethodRequest` object)
+
+        This handler is not compatible with PNP.
```

Comment:
I know we probably don't want to put PNP anywhere publicly. Should I refer to this as "Digital Twin" instead perhaps?